newsfragments/3326.deprecated.rst

```diff
@@ -0,0 +1 @@
+Both :class:`trio.testing.RaisesGroup` and :class:`trio.testing.Matcher` have been deprecated. Pytest alternatives pytest.RaisesGroup and pytest.RaisesExc (respectively) are considered correct replacement.
```

Comment:
You can use double backticks to have code formatting without requiring us to add an intersphinx thing for pytest